src/Aquifer.API/Telemetry/RequestTelemetryInitializer.cs

```diff
@@ -26,7 +26,7 @@ public void Initialize(ITelemetry telemetry)
 
         if ((httpContextAccessor.HttpContext?.Items.TryGetValue(Constants.HttpContextItemCachedApiKey, out var maybeCachedApiKey) ??
```

Comment:
I didn't update this to use the new property on `CachingApiKeyService` because this is probably the one time where the api-key might be null.